lib/plugins/extension/style.less

```diff
@@ -281,6 +281,15 @@
  * Enabled/Disabled overrides
  */
 #extension__list {
+    
+    .enabled, .disabled, .updatable {
+        display: none;
+    }
+    
+    &.enabled .enabled, &.disabled .disabled, &.updatable .updatable {
```

Comment:
Just a minor thing: Our CSS coding style is generally to have each selector on its own line (same for the block above).